csp/tests/test_jinja_extension.py

```diff
@@ -90,9 +82,6 @@ def test_regex_captures_script_content_including_brackets(self):
             {% endscript %}
             """
 
-        expected = (
-            '<script nonce="{}">'
-            'let capture_text = "<script></script>"'
-            '</script>')
+        expected = '<script nonce="{}">' 'let capture_text = "<script></script>"' "</script>"
```

Comment:
This looks like it could be cleaned up a little to avoid the `' ' " "` concatenation and quote type toggling. Same with the others in this file.